.gitignore

```diff
@@ -8,3 +8,4 @@ build/
 Build/
 venv/
 guitest.log
+jetbrains-core/src/software/aws/toolkits/telemetry/TelemetryDefinitions.kt
```

Comment:
remember to delete this

Reply:
Deleted now that it generates into build